public/data/Nnadivictory25.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "Nnadi  Victory",
```

There's an extra space in your name. I recommend removing it. What do you think? 😄
```suggestion
    "name": "Nnadi Victory",
```